Mage.Sets/src/mage/cards/d/DesmondMiles.java

```diff
@@ -0,0 +1,83 @@
+package mage.cards.d;
+
+import java.util.UUID;
+import mage.MageInt;
+import mage.abilities.Ability;
+import mage.abilities.common.DealsCombatDamageToAPlayerTriggeredAbility;
+import mage.abilities.common.SimpleStaticAbility;
+import mage.abilities.dynamicvalue.DynamicValue;
+import mage.abilities.dynamicvalue.common.AssassinsControlledAndInGraveCount;
+import mage.abilities.dynamicvalue.common.SavedDamageValue;
+import mage.abilities.dynamicvalue.common.StaticValue;
+import mage.abilities.effects.OneShotEffect;
+import mage.abilities.effects.common.continuous.BoostSourceEffect;
+import mage.abilities.effects.keyword.ScryEffect;
+import mage.abilities.effects.keyword.SurveilEffect;
+import mage.constants.*;
+import mage.abilities.keyword.MenaceAbility;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.game.Game;
+import mage.game.permanent.Permanent;
+
+/**
+ *
+ * @author grimreap124
+ */
+public final class DesmondMiles extends CardImpl {
+
+    public DesmondMiles(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.CREATURE}, "{1}{B}");
+        
+        this.supertype.add(SuperType.LEGENDARY);
+        this.subtype.add(SubType.HUMAN);
+        this.subtype.add(SubType.ASSASSIN);
+        this.power = new MageInt(1);
+        this.toughness = new MageInt(3);
+
+        // Menace
+        this.addAbility(new MenaceAbility());
+
+        // Desmond Miles gets +1/+0 for each other Assassin you control and each Assassin card in your graveyard.
+        Ability ability = new SimpleStaticAbility(new BoostSourceEffect(AssassinsControlledAndInGraveCount.FOR_EACH, StaticValue.get(0), Duration.WhileOnBattlefield));
```

Comment:
you don't need to make a custom value for this, just use `AdditiveDynamicValue` to combine two other values